CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.0.0-rc1 (October 22, 2015)
+### filter
+ - Fixed metrics filter to work with ES 2.0 changes which does not allow dots in field names
```

Comment:
I don't actually see this change in the diff between v2.0.0-beta3 and the 2.0 branch?

```
% git diff --numstat v2.0.0-beta3..HEAD
2       1       docs/asciidoc/static/breaking-changes.asciidoc
1       0       docs/asciidoc/static/contributing-to-logstash.asciidoc
2       7       docs/asciidoc/static/deploying.asciidoc
1       1       lib/logstash/version.rb
3       2       tools/Gemfile.beaker
```

No gemfile.lock change?